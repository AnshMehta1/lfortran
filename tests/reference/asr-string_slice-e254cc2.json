{
    "basename": "asr-string_slice-e254cc2",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/string_slice.f90",
    "infile_hash": "aaca6852e69f3ef20f618e36d1c1e75e52528c5ff757dd6d30538c13",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-string_slice-e254cc2.stderr",
    "stderr_hash": "ff3fac18ae1ddcc1c7533bf856571f4a906b6d1224619f33cead1447",
    "returncode": 2
}