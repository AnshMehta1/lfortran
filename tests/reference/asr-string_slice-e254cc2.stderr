semantic error: Substring `start` is negative
 --> tests/errors/string_slice.f90:4:13
  |
4 |     print*, s(-2:6)
  |             ^^^^^^^ 
